homeassistant/components/brother/sensor.py

```diff
@@ -104,10 +102,4 @@ def device_info(self):
         await self.printer.async_update()
 
         if self.printer.available:
```

Comment:
We don't need this check. The base entity will return `unavailable` as state when the entity is not available.

Reply:
Of course You're right :)